spec/models/legislation/process/phase_spec.rb

```diff
@@ -49,6 +49,11 @@
       process.update!(debate_start_date: Date.current - 2.days,
                       debate_end_date: Date.current - 1.day)
       expect(process.debate_phase.started?).to be true
+
+      # nil
+      process.update(!debate_start_date: nil,
```

Comment:
There is a syntax error here. Perhaps you meant to use `update!`?
```suggestion
      process.update!(debate_start_date: nil,
```

Reply:
Yep, my bad, thank you.